src/Symfony/Component/HttpClient/HttplugClient.php

```diff
@@ -61,6 +61,7 @@ final class HttplugClient implements ClientInterface, HttpAsyncClient, RequestFa
     private HttpClientInterface $client;
     private ResponseFactoryInterface $responseFactory;
     private StreamFactoryInterface $streamFactory;
+    private ?bool $honorRequestHttpVersion = null;
```

Comment:
naming is hard :)
WDYT of this?
```suggestion
    private bool $autoUpgradeHttpVersion = true;
```

Reply:
Great!